dev-util/trace-cmd/trace-cmd-3.2-r1.ebuild

```diff
@@ -61,6 +61,14 @@ pkg_setup() {
 	use python && python-single-r1_pkg_setup
 }
 
+src_prepare() {
+	default
+
+	# Not install libtracecmd library
```

Comment:
Let's include a bit more detail here about the weird scenario, so people do not wonder why this is being done in the future